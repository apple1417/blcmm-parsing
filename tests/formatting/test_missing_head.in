<BLCMM v="1">
#<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!>
	<body>
		<category name="root">
		</category>
	</body>
</BLCMM>

#Commands:
