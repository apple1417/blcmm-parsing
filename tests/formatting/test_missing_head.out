<BLCMM v="1">
	<body>
		<category name="root">
		</category>
	</body>
</BLCMM>
